.github/workflows/deploy-to-feature-environment.yml

```
@@ -53,6 +53,7 @@ jobs:
           echo "PR_NUMBER=${PR_NUMBER}" >> $GITHUB_ENV
 
       - name: Check for "🚀 Ready to deploy" label
+        if: contains(steps.check-labels.outputs.labels, '🚀 Ready to deploy')
```

Is this intentional?
```suggestion
        if: contains(steps.check-labels.outputs.labels, '🚀 Ready to deploy')
```